<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherForm</class>
 <widget class="QMainWindow" name="WeatherForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect to SQLite</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="dataTableView">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="populateButton">
      <property name="text">
       <string>Populate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sunnyDaysButton">
      <property name="text">
       <string>Sunny days</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="periodButton">
      <property name="text">
       <string>From a period</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="startDateEdit"/>
    </item>
    <item>
     <widget class="QDateEdit" name="endDateEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete tables</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
